### PR TITLE
Feature: Added progress bar to properties to indicate when the size is being calculated 

### LINKED ISSUE #8586: Feature: Add progress ring to properties when calculating sizes
### What's the Problem?

There used to be a loading icon when sizes were calculated which now doesnt appear and causes some confusion. See #8585

### Requirements
- Display progress ring next to the size labels to indicate when they are still loading

### Files Version

2.1.15.0

### Windows Version

_No response_

### Comments

_No response_

### PR DESCRIPTION
**Pull request by ferrariofilippo**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #8586

**Validation**
How did you test these changes?
- [x] Built and ran the app
- [x] Tested the changes for accessibility

**Screenshots (optional)**

### CODE REVIEW

**Comment by hecksmosis** [comment]
The spacing seems to be off, can you fix it? (Very slight issue)
image

**Comment by ferrariofilippo** [comment]
> The spacing seems to be off, can you fix it? (Very slight issue)

You mean vertically? Or do you mean it's not centered?

**Comment by yaira2** [comment]
Can you try to revert to the progress bar and see how it looks?

**Comment by ferrariofilippo** [comment]
> Can you try to revert to the progress bar and see how it looks?

Here it is:

https://user-images.githubusercontent.com/102259289/210447960-6e070777-020f-4af4-9b2e-1c47dbafd4af.mp4

**Comment by ferrariofilippo** [comment]
I think the `ProgressBar` looks better... what do you think? @yair100 @hecksmosis

**Comment by yaira2** [comment]
That looks really good

**Comment by QuaintMako** [comment]
I am more inline with the progress bar. The progress ring just feels out of place.

**Comment by hecksmosis** [comment]
The progress bar feels good

File: src/Files.App/Views/Pages/PropertiesGeneral.xaml [hunk 1]
```diff
@@ -176,10 +176,12 @@
 						HorizontalAlignment="Left"
 						Style="{StaticResource PropertyName}"
 						Text="{helpers:ResourceString Name=PropertiesItemSize/Text}"
+						VerticalAlignment="Center"
```
Comment (yaira2) [inline src/Files.App/Views/Pages/PropertiesGeneral.xaml:179]: This change needed?

**Review by yair100 — APPROVED** [review]
LGTM

**Pull request merged**